composer.json

```diff
@@ -148,6 +149,11 @@
         "symfony-web-dir": "web",
         "incenteev-parameters": {
             "file": "app/config/parameters.yml"
+        },
+        "heroku": {
+            "require": {
+                "ext-gd": "*"
```

Comment:
you should add ext-imagemagick to allow tests of ImageMagick adaptor (if they exists ?), what do you think ?